ansible/roles/baselayout/vars/main.yml

```diff
@@ -51,6 +51,10 @@ packages: {
     'gcc-c++,sudo',
   ],
 
+  ibmi: [
+    'gcc', 'gcc-cplusplus', 'gcc-cpp', 'ccache', 'sed-gnu', 'coreutils-gnu', 'zlib-devel', 'openssl-devel >= 1.1.1', 'python2-pip', 'git', 'ca-certificates-mozilla',
```

Comment:
Maybe do this as one item per line for legibility.

Reply:
Will do!